webcalc/templates/about.html

```diff
@@ -1,9 +1,153 @@
+        </p>
+
+        <ul>
+            <li>
+                <b>Unigram probability</b> (<tt>uni_prob</tt>): This is the standard unigram probability 
+
+                {% latexify 'P(w=x_1 \dots x_n) \approx \prod_{i=1}^{n} P(x_i)' math_block=True %}
+
+                where
+
+                {% latexify 'P(x) = \frac{C(x)}{\displaystyle\sum_{y \in \Sigma} C(y)}' math_block=True %}
+
+                where {% latexify 'C(x)' math_inline=True %} is the number of times the symbol {% latexify 'x' math_inline=True %} occurs in the training data.
+
+                <p>
+                    This metric reflects the probability of a word under a simple unigram model. The probability of a word is the product of the probability of its individual symbols. Note that the probability of the individual symbols is based only on their frequency of occurrence, not the position in which they occur.
+                </p>
+            </li>
+            <li>
+                <b>Bigram probability</b> (<tt>bi_prob</tt>): This is the standard bigram probability 
+
+                {% latexify 'P(w=x_1 \dots x_n) \approx \prod_{i=2}^{n} P(x_i|x_{i-1})' math_block=True %}
+
+                where
+
+                {% latexify 'P(x|y) = \frac{C(yx)}{C(y)}' math_block=True %}
+
+                where {% latexify 'C(y)' math_inline=True %} is the number of times the symbol {% latexify 'x' math_inline=True %} occurs in the training data and {% latexify 'C(yx)' math_inline=True %} is the number of times the sequence {% latexify 'yx' math_inline=True %} occurs in the training data. For bigram probabilities these counts are <i>Laplace Smoothed</i>: each possible bigram sequence begins with a count of 1, rather than 0. This means that bigram sequences not observed in the training data (that is, where {% latexify 'C(yx) = 0' math_inline=True %}) are treated as though they have been observed once, which gives them a small, rather than zero, probability.
+
+                <p>
+                    Each word is padded with a special start and end symbol, which allows us to calculate bigram probabilities for symbols that begin and end words.
+                </p>
+                <p>
+                    This metric reflects the probability of words under a simple bigram model. The probability of a word is the product of the probability of all the bigrams in contains. Note that the probability of the bigrams is based only on their frequency of occurrence, not the position in which they occur or their sequencing with respect to one another.
+                </p>
+            </li>
+            <li>
+                <p>
+                    <b>Positional unigram score</b> (<tt>pos_uni_prob</tt>): This is a type-weighted variant of unigram score from Vitevich and Luce (2004). 
+
+                    {% latexify 'PosUniScore(w=x_1 \dots x_n) \approx 1 + \sum_{i=1}^{n} P(w_i = x_i)' math_block=True %}
```

Comment:
Should we mention why the 1 is added or just leave it be since Vitevich and Luce explain it in their paper?